### PR TITLE
chore: fix server unit tests

### PR DESCRIPTION
**Pull request by ryanthemanuel**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/server/test/unit/fixture_spec.js
Comment (ryanthemanuel) [inline packages/server/test/unit/fixture_spec.js:164]:
```suggestion
      return Promise.map(Array(500), fn, { concurrency: 5 }).then(() => {
```

File: packages/server/test/unit/socket_spec.js
Comment (elevatebart) [inline packages/server/test/unit/socket_spec.js:53]: Why not use the standard Promise.all?

**Review by elevatebart — COMMENTED** [review]
Simple remark. Looks good! waiting on CI...

File: packages/server/test/unit/socket_spec.js
Comment (ryanthemanuel) [reply packages/server/test/unit/socket_spec.js]: Good catch. I was copying pasting from elsewhere. I missed a couple as well.

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
@ryanthemanuel Update base branch to be against develop before merging.

**Comment by ryanthemanuel** [comment]
@jennifer-shehane this PR is old and no longer valid.  Closing

**Pull request closed without merging**
